### PR TITLE
feat: monkeypatch sdk type and version

### PR DESCRIPTION
**Pull request by tore-statsig**
Statsig console has warning banners for updating out of date sdks.  Unfortunately, if you use the github sdk, it reports as statsig-node

This monkeypatches the library to report as the github sdk

Screenshot 2024-05-08 at 2 01 54 PM

### CODE REVIEW

File: src/__tests__/example.test.ts [hunk 1]
```diff
@@ -4,10 +4,10 @@ import { Inputs } from "../utils";
 test("example", async () => {
   const user = { userID: "test-local", email: "test-local@statsig.com" };
   const inputs: Inputs = {
-    sdkKey: process.env.TEST_SDK_KEY ?? "secret-",
+    sdkKey: "secret-Tpyw9SiCPwilEpeabun0kKRxiGaHHGQrHs9tbzzlGcy", //process.env.TEST_SDK_KEY ?? "secret-",
```
Comment (alex-statsig) [inline src/__tests__/example.test.ts:7]: Intentional?

**Review by alex-statsig — APPROVED** [review]
sgtm, long term maybe the client JS approach of exposing a `setStatsigPackageMetadata` (or something) function might be cleaner, but seems reasonable

File: src/__tests__/example.test.ts (hunk 1 above)
Comment (tore-statsig) [reply src/__tests__/example.test.ts:7]: rip

**Comment by tore-statsig** [comment]
Yeah I thought about that, implemented something overkill....and then gave up and went this route for now

**Pull request merged**